File: Lab05MadLips/app/src/main/java/com/example/lab05madlips/MainActivity.java
package com.example.lab05madlips;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    ImageView madLipLogo;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        madLipLogo = findViewById(R.id.madLipLogo);
        madLipLogo.setImageResource(R.drawable.madliblogo);

    }
}

File: Exams/app/src/main/java/com/example/exams/MainActivity.kt
package com.example.exams

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.speech.tts.TextToSpeech
import android.util.Log
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast
import java.util.*
import kotlin.math.log
import kotlin.math.log10

class MainActivity : AppCompatActivity() {

    private var tts: TextToSpeech? = null
    private var buttonSpeak: Button? = null
    private var editText: EditText? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        buttonSpeak = this.button_speak
        editText = this.edittext_input

        buttonSpeak!!.isEnabled = false;
        tts = TextToSpeech(this, this)
    }

    fun  onButton(view: View){
        etSpeak = findViewById(R.id.txt)
        btnSpeak = findViewById(R.id.voice)

        val text = etSpeak!!.text.toString()
        tts!!.speak(text, TextToSpeech.QUEUE_FLUSH, null,"")
    }
}